### ISSUE TITLE
JWChat: Support server connections

### ISSUE THREAD

**Issue by SunilMohanAdapa** [freedombox/freedombox-todos#31]
> Can cilents on the local server exchange messages with remote clients?

**Comment by NickDaly** [comment]
IIRC Jvalleroy had been looking into this.

James, can you comment on what we need for S2S XMPP communication?

**Comment by jvalleroy** [comment]
A few things are required:
1. Each box will need a domain name. I've tested both a dyn.com domain and a free domain from afraid.org.
2. The domain name should be added to ejabberd's hosts list, and set as the jwchat sitename. https://github.com/freedombox/Plinth/pull/118 will fix the configuration done by Plinth when the domain name is set.
3. I also found that enabling "s2s_use_starttls" in ejabberd.yml is necessary to talk to some other XMPP servers, such as jabber.org. I'll make a pull request on freedombox-setup to do this configuration during the setup.
